### ISSUE TITLE
AdminTools(Solo) don't work (anymore). Chance for fix?. 

### ISSUE THREAD

**Issue by 5andysalive** [Da0ne/DZMods#66]
Solo version doesn't work.
First it's  
127 function "onevent" is marked as override but there is no function with this name in the base class.

And without override
174  'this' or 'super' outside class is not valid.

Probably a minor isue for someone who knows what he's doing.  And i kno there is the better and bigger new onem but, I like the idea of a slim version even if it has less features. So a n00b proof fix would be great. thx.

**Comment by 5andysalive** [comment]
> Did you load DZmod?

no, only what the https://github.com/Da0ne/DZMods/blob/AdminTools(Solo)/README.md  said.

**Comment by modernkilla82** [comment]
DaOne no longer updates/supports the solo Admin Tools-
